src/main/resources/resources/logisim/strings/std/std_de.properties

```diff
@@ -55,6 +55,52 @@ dividerRemainderOutput = rem
 dividerRemainderTip = Rest: der Rest der Division (Dividend - Ausgang * Divisor)
 dividerUpperInput = upper
 #
+# arith/FPAdder.java
```

Comment:
Is sync of properties files related to your PR? If not, can you please roll it back (multiple places).

Reply:
I added two new keys and used the syncTool. There were even more syncs in different files, but I included only those where I added something.